types/rlp_marshal.go

```diff
@@ -190,14 +191,15 @@ func (t *Transaction) MarshalRLPTo(dst []byte) []byte {
 
 // MarshalRLPWith marshals the transaction to RLP with a specific fastrlp.Arena
 func (t *Transaction) MarshalRLPWith(arena *fastrlp.Arena) *fastrlp.Value {
+	fmt.Println("***** parts", 13372, t.Nonce, t.GasTipCap, t.GasFeeCap, t.Gas, t.To, t.Value, t.Input, t.Type, t.V, t.R, t.S)
 	vv := arena.NewArray()
 
 	// Specify zero chain ID as per spec.
 	// This is needed to have the same format as other EVM chains do.
 	// There is no chain ID in the TX object, so it is always 0 here just to be compatible.
 	// Check Transaction1559Payload there https://eips.ethereum.org/EIPS/eip-1559#specification
 	if t.Type == DynamicFeeTx {
-		vv.Set(arena.NewBigInt(big.NewInt(0)))
+		vv.Set(arena.NewBigInt(big.NewInt(13372)))
```

Comment:
I am not sure what these changes are.

Reply:
I'm sorry @goran-ethernal. That was a bad push 🤦 Can you have another look?